crates/l2/sequencer/errors.rs

```diff
@@ -159,6 +161,10 @@ pub enum StateDiffError {
     LengthTooBig(#[from] core::num::TryFromIntError),
     #[error("DB Error: {0}")]
     DbError(#[from] TrieError),
+    #[error("Store Error: {0}")]
+    StoreError(#[from] StoreError),
+    #[error("StateDiff registered a negative nonce in AccountUpdate")]
```

Comment:
This name is a little bit confusing, i had to go to the code to interpret it. From what i inferred, it would mean than the nonce didn't change. I think that something the following msg sounds better:
```suggestion
    #[error("StateDiff didn't register a nonce change while calculating the AccountUpdate")]
```

Reply:
It means the new nonce is lower than the previous one. I'll update it accordingly!

Reply:
Addressed in [651ed90](https://github.com/lambdaclass/ethrex/pull/2292/commits/651ed903f65b4ed9ea56cacfd09cb5ef58eb30f0)!